data/json/npcs/TALK_CYBORG_1.json

```diff
@@ -2,10 +2,169 @@
   {
     "id": "TALK_CYBORG_1",
     "type": "talk_topic",
-    "dynamic_line": "You... You saved me!  *Zzzt* Let me come with you, I can help!",
+	"dynamic_line":
+	{
+        "npc_has_var": "cyborg_has_talked",
+		"type": "dialogue",
+		"context": "cyborg",
+		"value": "yes",
+		"no": "I... I'm free. *Zzzt* I'm actually free! *bzzz* Look, you're the first person I've seen in a long time.",
+		"yes": "Hey again. *kzzz*"
+	},
+  
     "responses": [
-      { "text": "You're on your own.  Bye.", "topic": "TALK_DONE" },
-      { "text": "Of course!  Come with me.", "topic": "TALK_AGREE_FOLLOW" }
+	  {
+        "switch": true,
+		"condition": { "npc_has_var": "cyborg_has_talked", "type": "dialogue", "context": "cyborg", "value": "yes" },
+		"text": "Hey. Let's chat for a second.",
+		"topic": "TALK_STRANGER_NEUTRAL"
+	  },
+      {
+        "switch": true,
+        "default": true,
+        "text": "That's because I saved you. Listen, I could use your help...",
+        "trial": { "type": "PERSUADE", "difficulty": 0 },
+        "success": { "topic": "TALK_CYBORG_FRIENDLY", "opinion": { "trust": 1, "value": 1 }, "effect": { "npc_add_var": "cyborg_has_talked", "type": "dialogue", "context": "cyborg", "value": "yes" } },
+        "failure": { "topic": "TALK_CYBORG_WARY", "opinion": { "anger": 1, "fear": 1 }, "effect": { "npc_add_var": "cyborg_has_talked", "type": "dialogue", "context": "cyborg", "value": "yes" } }
+      },
+      {
+        "switch": true,
+        "default": true,
+        "text": "Big fucking deal. Do what I say or I'll shove that chip back where it belongs.",
+        "trial": { "type": "INTIMIDATE", "difficulty": 20, "mod": [ [ "AGGRESSION", -2 ], [ "BRAVERY", -2 ] ] },
+        "success": { "topic": "TALK_CYBORG_FEARFUL", "opinion": { "trust": -4, "fear": 3, "value": -1, "anger": 4 }, "effect": { "npc_add_var": "cyborg_has_talked", "type": "dialogue", "context": "cyborg", "value": "yes" } },
+        "failure": { "topic": "TALK_CYBORG_BREAKDOWN", "opinion": { "trust": -4, "value": -5, "anger": 10 }, "effect": { "npc_add_var": "cyborg_has_talked", "type": "dialogue", "context": "cyborg", "value": "yes" } }
+      },
+      {
+        "switch": true,
+        "default": true,
+        "text": "Sorry, I'm nobody. Enjoy your freedom, I guess.",
+		"effect": { "npc_add_var": "cyborg_has_talked", "type": "dialogue", "context": "cyborg", "value": "yes" },
+        "topic": "TALK_DONE"
+      }
     ]
+  },
+  {
+    "id": "TALK_CYBORG_FRIENDLY",
+	"type": "talk_topic",
+	"dynamic_line": "*buzz* Great! So what happens now?",
+    "responses": [
+      {
+        "switch": true,
```

Comment:
you don't need a switch and a default in both response objects.  If you're not intending to short circuit the process, you don't need a switch.